repository-data/application/src/main/resources/hcm-config/configuration/queries/templates/new-national-indicator-library-folder.yaml

```diff
@@ -0,0 +1,30 @@
+---
```

Comment:
You need to apply this to the NIL folder right?

Reply:
I think I have this correct now - certainly it seems to work fine locally.